app/client/survey_admin/survey_admin_users.jade

```diff
@@ -1,2 +1,3 @@
 template(name="survey_admin_users")
-  h3 Users
+  .survey-admin--content
```

Comment:
I'm going for something similar to the BEM (block, element, modifier) naming convention. Normally the name that follows `--` is suggestive of a modifier but I'm suggestion a sub-element. I think BEM recommends `__` for elements but I think some only use `--`. Trying to add a little more structure to the CSS.